pkg/store/store.go

```diff
@@ -572,7 +573,7 @@ func (store *OpenTSDBStore) checkMetricNames(metricNames []string, fullBlock boo
 func (store *OpenTSDBStore) convertOpenTSDBResultsToSeriesResponse(respI *opentsdb.QueryRespItem) (*storepb.SeriesResponse, int, error) {
 	name := respI.Metric
 	if store.enableMetricNameRewriting {
-		name = strings.ReplaceAll(strings.ReplaceAll(name, ".", ":"), "-", "_")
```

Comment:
Could we do a switch for that? Defaults to the old behaviour, but allows someone who hasn't used Geras before to make everything bad come back as `_`?

Reply:
Yes, seems reasonable.

Reply:
Note one downside of this behaviour is it becomes difficult to unit test your rules, i.e. a query for `tsd:rpc:errors` would currently return `tsd:rpc:errors`, which means in a promtool unit test you can simply have that input series.

As soon as there's any rewriting happening between name that is queried and name that is returned it's not possible to unit test (we had very few metrics with a `-` so that was reasonable, but all our metrics have a `.` in).